card/mod/05_standard/set/rstar/rules.rb

```diff
@@ -1,9 +1,10 @@
+
 format :html do
 
   view :closed_rule, :tags=>:unknown_ok do |args|
     return 'not a rule' if !card.is_rule? #these are helpful for handling non-rule rstar cards until we have real rule sets
       
-    rule_card = card.new_card? ? find_current_rule_card[0] : card
+    rule_card = card.new_card? ? find_current_rule_card : card
```

Comment:
It changed. It returns only the current_rule_card now and not an array with additional stuff.